### PR TITLE
xt-only yaml file added as an example of how to connect to an XT SEM

### PR DESCRIPTION
**Pull request by tepals**

### CODE REVIEW

File: install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+"EBeam control": {
+  role: e-beam,
+  init: {
+    hfw_nomag: 0.11,
+  },
+}
+
+"EBeam Focus": {
+  role: focus,
```
Comment (pieleric) [inline install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml:30]: The role should be `ebeam-focus`.

File: install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+"EBeam Stage": {
+  role: stage,
+  init: {
+    inverted: ["x", "y"],
+  }
+}
```
Comment (pieleric) [inline install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml:38]: Please add a `affects: ["EBeam Scanner"],`
This allow to have MD_POS automatically updated, which means that the SEM image will appear at the right place.

File: install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  children: {
+    scanner: "EBeam control",
+    focus: "EBeam Focus",
+    stage: "EBeam Stage",
+    detector: "EBeam Detector",
+  }
+}
+
+"EBeam control": {
```
Comment (pieleric) [inline install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml:22]: Let's call this "EBeam Scanner" (with the uppercase).

File: install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+SEM: {
```
Comment (pieleric) [inline install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml:1]: Could you add somewhere as a comment that the Support PC should be running xtadapter?

**Review by pieleric — CHANGES_REQUESTED** [review]

File: install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml [hunk 5]
```diff
@@ -1,40 +1,43 @@
...
 "EBeam Focus": {
-  role: focus,
+  role: ebeam-focus,
 }
 
 "EBeam Stage": {
   role: stage,
   init: {
     inverted: ["x", "y"],
   }
```
Comment (pieleric) [inline install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml:39]: You're missing a comma.
Please test it one last time before we merge it ;-)

File: install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
+SEM: {
+  class: Microscope,
+  role: sem,
+  children: ["SEM EXT", "EBeam Scanner", "EBeam Focus", "EBeam Stage",
```
Comment (pieleric) [inline install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml:4]: Actually, the "SEM EXT" doesn't need to be a child of the sem, as it plays no role (directly).

**Review by Sabrina85 — APPROVED** [review]
Looks fine (except for Eric's last 2 comments). I tested it quickly on EA1 and it works (without the stage but with the detector).

**Pull request merged**